doc/src/manual/performance-tips.md

```diff
@@ -1239,6 +1239,34 @@ versus:
 println(file, f(a), f(b))
 ```
 
+## Avoid eager string materialization
+
+In settings where a string representation of an object is only needed
+conditionally (e.g. in error paths of functions or conditional warnings such as
+deprecations), it is advisable to avoid the overhead of eagerly materializing
+the string. Since Julia 1.8, this can be achieved with via
```

Comment:
I think you can remove either 'with' or 'via'.

Reply:
Good catch!

Reply:
Fixed